### DISCUSSION TITLE
/fate-blog/%E5%AE%89%E8%A3%85%E6%95%99%E7%A8%8B/%E5%AE%89%E8%A3%85JDK.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#325] [Announcements]
# /fate-blog/%E5%AE%89%E8%A3%85%E6%95%99%E7%A8%8B/%E5%AE%89%E8%A3%85JDK.html

vuepress-theme-hope 的演示

http://xuhai.press/fate-blog/%E5%AE%89%E8%A3%85%E6%95%99%E7%A8%8B/%E5%AE%89%E8%A3%85JDK.html

**Comment by 15250020yourself** [comment]
test
